schemas/littlehorse/service.proto

```diff
@@ -80,6 +81,15 @@ service LittleHorse {
   // Runs a WfSpec to create a WfRun.
   rpc RunWf(RunWfRequest) returns (WfRun) {}
 
+  // Schedule repeated WfRun based on a cron expression
+  rpc ScheduleWf(ScheduleWfRequest) returns (ScheduledWfRunId) {}
```

Comment:
Can we have it `returns (ScheduledWfRun) {}`? 

The `rpc PutExternalEvent` returns `ExternalEvent`, not `ExternalEventId`. Same with `rpc RunWf`.